File: 백준/Bronze/2576. 홀수/홀수.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace MyCompiler {
    class Program {
        public static void Main(string[] args) {

            int playCnt = 0;
            List<int> numbers = new List<int>();
            
            while (playCnt < 7)
            {
                // 하나의 숫자를 입력받아 int로 변환
                int num = int.Parse(Console.ReadLine());
                
                // 홀수인 경우에만 리스트에 추가
                if (num % 2 != 0) 
                    numbers.Add(num);

                playCnt++; // playCnt 증가
            }

            // 오름차순 정렬
            numbers = numbers.OrderBy(value => value).ToList();

            if(numbers.Count > 0)
            {
                int sum = 0;
            
                // 홀수 배열 합 구하기
                for(int i = 0; i < numbers.Count; i++) sum += numbers[i];

                Console.WriteLine($"{sum}\n{numbers[0]}");    
            }

            else Console.WriteLine(-1);
            
        }
    }
}

File: 백준/Bronze/2576. 홀수/홀수.cc
#include <bits/stdc++.h>

using namespace std;

int main() {

    int playCnt = 0;

    vector<int> numbers;

    // 7개 숫자 입력받기
    while( playCnt < 7)
    {
        int num = 0;

        cin >> num;

        // 홀수인지 확인하기
        if( num != 0 && num % 2 != 0) numbers.push_back(num);

        // 횟수 증가
        playCnt++;
    }

    // 홀수 값 없었으면 -1 반환하기
    if(numbers.size() == 0)
    {
        cout << -1;
        return 0;
    }

    // 홀수 값 있으면 핪과 최솟값 반환하기ㅎ
    else
    {
        int sum = 0;
        
        sort(numbers.begin(), numbers.end());
        
        for(int i = 0; i < numbers.size(); i++) sum += numbers[i];

        cout << sum << '\n'<< numbers[0];

        return 0;
    }
}